src/GitLinkTask/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "NuProj.Common": "0.11.14-beta"
```

Comment:
@GeertvanHorrik I have used NuProj before.  From a purely personal view point, I prefer the creation of the NuGet Packages to be controlled via some sort of build script, for example Cake.  But obviously, I am slighly biased here 😄 

@AArnott do you know if this works when building on another environment, such as XBuild on Linux?